include/sentry.h

```diff
@@ -561,7 +561,6 @@ SENTRY_API void sentry_envelope_free(sentry_envelope_t *envelope);
 SENTRY_API sentry_value_t sentry_envelope_get_event(
     const sentry_envelope_t *envelope);
 
-#ifdef SENTRY_PERFORMANCE_MONITORING
```

Comment:
I think this one is now unbalanced?

Reply:
thanks for catching this! looks like i didn't remove this one's partner `#endif`, fixed this.